steps/install-flutter-version-manager/1.0.0/step.yml

```diff
@@ -0,0 +1,33 @@
+title: Install Flutter Version Manager
+summary: |
+  Installs Flutter Version Manager for the project
+description: |
+  Installs FVM (Flutter Version Manager) for the project and replaces Flutter version so that other Bitrise steps can use it
+website: https://github.com/infinum/Install-Flutter-Version-Manager-Bitrise
+source_code_url: https://github.com/infinum/Install-Flutter-Version-Manager-Bitrise
+support_url: https://github.com/infinum/Install-Flutter-Version-Manager-Bitrise/issues
+published_at: 2020-12-08T10:41:36.723477+01:00
+source:
+  git: https://github.com/infinum/Install-Flutter-Version-Manager-Bitrise.git
+  commit: 2321314493754681dbd4c366b2b63102e0a8c08c
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- flutter
+type_tags:
+- installer
+toolkit:
+  bash:
+    entry_file: step.sh
```

Comment:
You are using macOS specific path in source but listing linux in host_os_tags.

Reply:
Oh. Good catch. I'll modify it to include only macOS. The flutter runs on macOS since there's no way to build iOS apps on linux anyway.